action.yml

```diff
@@ -238,6 +238,10 @@ runs:
       if: ${{ inputs.phplinting == 'true' }}
       shell: bash
       run: |
+        if [[ -f vendor/bin/parallel-lint ]]; then
+          echo "Running parallel-lint"
+          vendor/bin/parallel-lint --exclude=vendor --exclude=.git .
```

Comment:
Excluding `.git` is just for speed - means it doesn't have to look at all the files in there checking for php files that aren't there.